### ISSUE TITLE
CV-04. Обучить второй независимый image-кандидат

### ISSUE THREAD

**Issue by mephiacc-david** [Bersserker/hakathon_room_cv#10]
Что нужно сделать: поднять вторую модель другой семьи и/или dual-view full image + smart-crop branch, чтобы ошибки отличались от первой модели.
Артефакт: artifacts/oof/model2_v1/, reports/model2_eval_v1.md → передает У1, У4, У5.
Зависимости: stable train loop, smart-crop policy от У4/У2, benchmark по compute.
DOR: baseline recipe заморожен; есть свободное compute window.
DOD: второй кандидат полностью обучен на development split, имеет OOF/shadow метрики и стандартный export logits.
Оценка трудозатрат: 1.5 дн
Приоритет: P1

**Comment by mephiacc-david** [comment]
CV-04: обучить вторую image-модель

Цель: другая модель, чтобы ошибки отличались.

Сейчас baseline:

backbone: convnext_tiny.in12k_ft_in1k
image_size: 224

Для CV-04 лучше взять другую семью из whitelist. Сейчас whitelist:

whitelist:

convnext_tiny.in12k_ft_in1k
efficientnet_b0
resnet50

Самый простой вариант:

backbone: efficientnet_b0

или:

backbone: resnet50

Без dual-view пока. Dual-view сложнее, в коде нет поддержки smart-crop branch. Для 1.5 дня проще взять EfficientNet-B0 или ResNet50 как model2.

──────────────────────────────────────────────────────────

Что сделать для CV-04

Создать config:

configs/model/model2_v1.yaml

На базе baseline, но:

model:
backbone: efficientnet_b0

И:

artifacts:
oof_dir: artifacts/oof/model2_v1
report_path: reports/model2_eval_v1.md

experiment:
version: v01
loss: ce
sampler: shuffle
ratio_policy: none
weak_label_flag: false
feature_flags: img_model2
tta_flag: false

Запуск:

uv run python src/training/train_image.py --config configs/model/model2_v1.yaml --all-folds

Результат должен появиться:

artifacts/oof/model2_v1/oof_predictions.parquet
artifacts/oof/model2_v1/shadow_holdout_predictions.parquet
reports/model2_eval_v1.md

──────────────────────────────────────────────────────────

Главное простыми словами

На чем учить?

На уже готовом split:

data/splits/splits_v1.json

Train folds внутри него.
Shadow holdout не трогать для обучения.
